docs/mint.json

```diff
@@ -286,6 +280,17 @@
       "api-reference/logs/post-logs"
     ]
   },
+  {
+    "group": "Org",
```

Comment:
Consider adding cross-references between related client and organization API endpoints in the documentation to help users understand the relationship between them.